### PR TITLE
Release changelog for 2.12

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+  [ Simon Quigley ]
+  * Change apt-get to apt in HACKING.md (#564) (LP: #1591421)
+  * Changed the mailing list in HACKING.md from snappy-devel to snapcraft
+    (#577)
+  * Add Subversion support (#567) (LP: #1543243)
+
+  [ Joe Talbott ]
+  * Include 'maintainer' and 'description' in the parser output. (#565)
+    (LP: #1591199)
+  * Support "```" wiki code tags in the parser. (#569)
```
Comment (elopio) [inline debian/changelog]: LP: #1592133

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,62 @@
...
+  * Add snapcraft-parser integration test. (#560) (LP: #1590268)
+  * Make most wiki fields required.  Bugs/1593390 (#581)
```
Comment (elopio) [inline debian/changelog]: Wrong format for the bug.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,62 @@
...
+  * Add more info about reusable parts. (#527) (LP: #1582499)
+  * Allow parts without a 'source' entry. (#599)
```
Comment (elopio) [inline debian/changelog]: LP: #1595666

File: debian/changelog (hunk 2 above)
Comment (kyrofa) [inline debian/changelog]: Might want to reformat this one.

File: debian/changelog (hunk 3 above)
Comment (kyrofa) [inline debian/changelog]: LP: #1595666

**Comment by elopio** [comment]
:+1: 
We are starting to get new contributors, so I have a suggestion for the release announcement. Add a section to thank people with their first accepted change in the project.

**Comment by sergiusens** [comment]
El 27/06/16 a las 14:14, Leo Arias escribió:
> 👍
> We are starting to get new contributors, so I have a suggestion for the
> release announcement. Add a section to thank people with their first
> accepted change in the project.
Good idea!

**Comment by kyrofa** [comment]
> Add a section to thank people with their first accepted change in the project.

That would be tough to track. Great idea though-- I suggest we just always thank contributors who aren't part of the core group, even if they've contributed multiple times.

**Pull request merged**